src/insertion.py

```diff
@@ -0,0 +1,41 @@
+"""Create insertion sort."""
+
+
+def insertion_sort(potato):
```

Comment:
Please don't call the input list "potato". At a glance, I can't tell what you're sorting over